prestapyt/dict2xml.py

```diff
@@ -21,15 +21,15 @@ def _process(doc, tag, tag_value):
     @param tag_value: tag value
     @return: node or nodelist, be careful
     """
-    if isinstance(tag_value, dict) and 'value' in tag_value.keys() == ['value']:
+    if isinstance(tag_value, dict) and 'value' in list(tag_value.keys()) == ['value']:
         tag_value = tag_value['value']
 
     if tag_value is None:
         tag_value = ''
 
     # Create a new node for simple values
     if (isinstance(tag_value, (float, int)) or
-            isinstance(tag_value, (str, unicode))):
+            isinstance(tag_value, str)):
```

Comment:
This will break compatibility with python2, you can add the `future` library and use this pattern:

```
from past.builtins import basestring

... isinstance(tag_value, basestring)
```